alvr/server/cpp/alvr_server/Settings.cpp

```diff
@@ -105,10 +105,15 @@ void Settings::Load() {
         m_enableControllers = config.get("controllers_enabled").get<bool>();
         m_controllerIsTracker = config.get("controller_is_tracker").get<bool>();
 
+        auto sessionSettings = v.get("session_settings");
+        auto sessionVideo = sessionSettings.get("video");
+
+        m_h264UseBaselineProfile = sessionVideo.get("h264_use_baseline_profile").get<bool>();
```

Comment:
If you write this code like this, SteamVR will not restart when this setting has been changed. Please check how the other settings are passed to SteamVR and do something similar.